docs/testplan/BGP_IPv6_test_2TierNetwork.md

```diff
@@ -0,0 +1,32 @@
+# Test Objective
+This test aims to verify the scalability and stability of 256 BGP sessions and 10K IPv6 routes in a 2-tier network. It evaluates the DUT’s ability to establish and maintain BGP sessions, ensures proper route learning, and measures BGP update convergence time under various conditions.
+
+# Test Setup
+![Test Setup](./2TierNetwork.png)
+
+1.	The testbed consists of four IXIA traffic generators (synchronized using a time-sync metronome) and five SONiC switches, where the BT1 switch is the Device Under Test (DUT).
+2.	Each of the four BT0 switches is connected to the DUT via eight DAC cables. There are no direct connections between any two BT0 switches.
+3.	Each BT0 switch is also connected to one IXIA traffic generator via eight optical cables. Similarly, there are no direct connections between any two IXIA devices.
+4.	Both switches and IXIAs support four port breakout modes: 8x100Gbps, 4x200Gbps, 2x400Gbps, and 1x800Gbps. However, they must operate in the same mode. In 8x100Gbps mode, each cable supports eight links. In 4x200Gbps mode, each cable supports four links. So on and so forth.
+5.	The routing configuration of the BT0 switches should ensure that all data traffic go through the DUT.
+
+# Test Steps
+1. Assign a unique AS number to each of the five switches.
```

Comment:
we should consider automate this using add-topo or deploy-mg.

all switches can share the same setup, and ask the IXIA to advertise the routes.